_assets/stylesheets/components/_modules.scss

```diff
@@ -2,5 +2,8 @@
 @import "./cards";
 @import "./locations-grid";
 @import "./preloader";
-@import "./utilities";
```

Comment:
I could be wrong but I think we may have been using some of the utilities styles.

Reply:
This file was moved outside the components directory since these styles aren't component based.